compliance/collection/main.go

```diff
@@ -183,14 +186,15 @@ func manageNodeScanLoop(ctx context.Context, i intervals.NodeScanIntervals, scan
 	return sensorC
 }
 
-func scanNode(nodeName string, scanner nodeinventorizer.NodeInventorizer) (*sensor.MsgFromCompliance, error) {
-	result, err := scanner.Scan(nodeName)
+func scanNode(scanner scannerV1.NodeInventoryServiceClient) (*sensor.MsgFromCompliance, error) {
+	result, err := scanner.GetNodeInventory(context.Background(), &scannerV1.GetNodeInventoryRequest{})
```

Comment:
Do we need that request param if it is always a zero-value? Could we maybe hide it inside `scanner.GetNodeInventory`?

Reply:
IIRC, this is a recommendation when defining Protobuf services.
By defining the service with a `Request` from the beginning, we don't break backwards compatibility if we ever introduce request parameters.
`Get NodeInventory` is already the generated protobuf function, so I don't know whether we would like to wrap that in another layer.

Reply:
Oh, so if `GetNodeInventory` is a proto-generated func, then all looks good.